modules/programs/i3status-rust.nix

```diff
@@ -258,8 +251,8 @@ in {
         '';
 
         source = settingsFormat.generate ("config-${cfgFileSuffix}.toml") ({
-          theme = cfg.theme;
-          icons = cfg.icons;
+          theme = { theme = cfg.theme; };
+          icons = { icons = cfg.icons; };
```

Comment:
We could make this change backwards compatible if we check the version of `i3status-rust` here.

Reply:
Maybe check for the `home.stateVersion` here?

Reply:
Done in fb3dff74d14c9536317d83854545df6556244a44.